scenarios/kubernetes_e2e.py

```diff
@@ -442,6 +459,9 @@ def create_parser():
         '--test', default='true', help='If we need to set --test in e2e.go')
     parser.add_argument(
         '--up', default='true', help='If we need to set --up in e2e.go')
+    parser.add_argument(
+        '--multiple-federations', default=False, action='store_true',
```

Comment:
That feels like arbitrary aliasing of environment variables.

Moreever, how do I tell kubetest that I need this flag to be activated? Pass that as an environment variable in my test env file and enable that flag in `e2e-runner.sh`?